logic.py

```
@@ -566,6 +577,32 @@ def count_val(series: pd.Series, v_str: str) -> int:
+                 # For linear, results_db[col] has 'p_or' inside 'or_results' logic? 
+                 # Wait, logic is complex. 'p_or' key exists in 'res' for linear.
+                 if 'p_or' in res and isinstance(res['p_or'], (float, int)):
+                     p = res['p_or']
+            
+            if pd.notna(p) and isinstance(p, (float, int)):
+                uni_p_vals.append(p)
+                uni_keys.append(col)
+        
+        if uni_p_vals:
+            adj_p = apply_mcc(uni_p_vals, method=mcc_method)
+            for k, p_adj in zip(uni_keys, adj_p):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)